### PR TITLE
Codecov config tweaks

### PR DESCRIPTION
**Pull request by swissspidy**
Need to re-add the Codecov GitHub app first (already requested).

### CODE REVIEW

File: codecov.yml
Comment (barklund) [inline codecov.yml:19]: What does it do?

Comment (swissspidy) [reply codecov.yml:19]: Too many things 😅 

One part is for tweaking the PR comment made by the bot, making sure that it only works for PR targeting master.

The other part is for the Codecov GitHub app that can actually prevent merging PRs when coverage drops significantly. Although I disabled that particular functionality with the `informational: true` flag.

**Review by barklund — APPROVED** [review]
Let's try it out.

**Comment by swissspidy** [comment]
Alright. As mentioned in the PR description I have to wait for the app approval, but that should be coming soon I suppose.

**Pull request merged**
